test/globals-test.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
what error are you getting if you remove this line?

Reply:
Nothing, I revert this change. I am thinking that I added  it when I don't install fake-indexdb